### ISSUE TITLE
MiKo_2020 should report only if linked documentation can be inherited

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#463]
Today, MiKo_2020 reports a violation for e.g. following code:
```c#
/// <summary>
/// See <see cref="string.Format"/>
/// </summary>
public void Bla() { }
```
In such case (reference in `<see>` is not inherited/implemented), MiKo_2020 should not report an issue.
